solidity/dashboard/src/components/LiquidityRewardCard.jsx

```diff
@@ -70,24 +79,39 @@ const LiquidityRewardCard = ({
       <div className={"liquidity__info text-grey-60 mt-2"}>
         <div className={"liquidity__info-tile bg-mint-10"}>
           <h2 className={"liquidity__info-tile__title text-mint-100"}>
-            {formattedApy}*
+            {formattedApy.value === Infinity ? (
+              <span>&#8734;</span>
+            ) : (
+              <CountUp
+                end={formattedApy.value}
+                // Save previously ended number to start every new animation from it.
+                preserveValue
+                decimals={2}
+                formattingFn={(value) => `${formattedApy.prefix}${value}%`}
+              />
+            )}
           </h2>
           <h6>Annual % yield (APY)</h6>
         </div>
         <div className={"liquidity__info-tile bg-mint-10"}>
           {isFetching ? (
             <Skeleton tag="h2" shining color="mint-20" />
           ) : (
-            <h2
-              className={"liquidity__info-tile__title text-mint-100"}
-            >{`${formattedPercentageOfTotalPool}%`}</h2>
+            <h2 className={"liquidity__info-tile__title text-mint-100"}>
+              <CountUp
+                end={formattedPercentageOfTotalPool.value}
+                // Save previously ended number to start every new animation from it.
+                preserveValue
+                decimals={2}
+                formattingFn={(value) =>
+                  `${formattedPercentageOfTotalPool.prefix}${value}%`
+                }
+              />
```

Comment:
I would consider lowering the duration of the animation to 1 second (I believe default is 2 seconds). The user will probably have to wait until the data is loaded on the mainnet and then additional 2 seconds for the animation to finish - seems a little bit too long to me and may upset some of the users in the long run 😄 But who knows, maybe i'm the only one impatient here.

Anyway to lower the duration you just have to add duration={1}. I would do that in all occurrences of CountUp if you agree with my point of view.